tests/test_processing_stack.py

```diff
@@ -483,10 +511,9 @@ def should_successfully_run_dataset_version_creation_process_and_again_with_part
 
             #  Wait for Item metadata file to include a link to geostore root catalog
             expected_link_object = {
-                STAC_HREF_KEY: f"../{CATALOG_FILENAME}",
+                STAC_HREF_KEY: CATALOG_FILENAME,
```

Comment:
I don't think this is right. The root should always be the actual root catalogue file, which is in the parent directory of the dataset.

Reply:
You are correct about this. Thanks for flagging it. The problem was caused by calling `update_root_link` before another function which `change_href_to_basename`. Took some time to figure out, but makes more sense now.